fixtures/topology/k8s-system.yaml

```diff
@@ -36,6 +36,10 @@ spec:
                 name: nodemetrics
                 display:
                   javascript: JSON.stringify(k8s.getNodeMetrics(results))
+        - name: cluster-name
+          configLookup:
+            display:
+              javascript: config.getConfigItem("EKS/flanksource-canary-cluster").properties["name"]
```

Comment:
GetConfigItem by type/name is useful, but the primary use case would be to get the config item related to the component

Reply:
configItemByType("EKS") -> [array ConfigItems]
configGetItem()